forc/src/ops/forc_fmt.rs

```diff
@@ -152,3 +177,67 @@ impl From<io::Error> for FormatError {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::taplo_fmt;
+    use std::default::Default;
+
+    #[test]
```

Comment:
Are there tests for alphabetization? If the formatter doesn't support alphabetization of imports and fields, can that be added? It can also be reserved for a future PR.

Reply:
I can add tests for alphabetization, I will add the corresponding option in the taplo formatter as well